### DISCUSSION TITLE
Games on DeepIn: StarCraft 2 and Wine in DeepIn v23

### DISCUSSION THREAD

**Discussion by ADalles** [linuxdeepin/developer-center#10524] [Troubleshooting | 故障排查]
Hi!

Thanks for your beautiful OS based on Linux. I try start game of Starcraft II but it is failed.

My devices:
CPU AMD Ryzen 1600
GPU AMD Radeon HD 7970
Memory 64 Gb
OS: DeepIn v23

I try install Battle.net but after success install this application not running.
I had this error

> deepin-wine8-stable Games/Battle.net/Battle.net.exe 
> wine version: 8.16
> ac_compute_device_uuid's output is based on invalid pci bus info.
> ac_compute_device_uuid's output is based on invalid pci bus info.
> [0830/143221.027:ERROR:network_change_notifier_win.cc(143)] WSALookupServiceBegin failed with: 0
> [0830/143221.127:ERROR:network_sandbox.cc(302)] Failed to grant sandbox access to cache directory C:\users\alles\AppData\Local\Battle.net\BrowserCaches\common\C
> ache\Cache_Data: ╧ЁюЎхфєЁр эх эрщфхэр. (0x7F)
> [0830/143221.127:ERROR:network_sandbox.cc(396)] Failed to grant sandbox access to network context data directory C:\users\alles\AppData\Local\Battle.net\Browser
> Caches\common\Network: ┬√яюыэхэю єёях°эю. (0x0)
> [0830/143221.130:ERROR:network_service_instance_impl.cc(270)] Encountered error while migrating network context data or granting sandbox access for C:\users\all
> es\AppData\Local\Battle.net\BrowserCaches\common\Network. Result: 6: ┬√яюыэхэю єёях°эю. (0x0)
> ac_compute_device_uuid's output is based on invalid pci bus info.
> ac_compute_device_uuid's output is based on invalid pci bus info.
> ac_compute_device_uuid's output is based on invalid pci bus info.
> wine: Unhandled exception 0x80000003 in thread 374 at address 1482B0E1 (thread 0374), starting debugger...

and log Battle.Net is

> 0x1482b0e1 libcef+0x16d00e1: int3
> Modules:
> Module	Address			Debug info	Name (79 modules)
> PE	00400000-004f6000	Export          battle.net
> PE	005e0000-00a28000	Deferred        ole32
> PE	00a30000-00c39000	Deferred        rpcrt4
> PE	00c40000-00f3b000	Deferred        crypt32
> PE	025e0000-02646000	Deferred        msacm32
> PE	02650000-02769000	Deferred        dxgi
> PE	02770000-03973000	Deferred        wined3d
> PE	03980000-03c11000	Deferred        dwrite
> PE	03c20000-03c22000	Deferred        usp10
> PE	03c30000-03e9d000	Deferred        urlmon
> PE	03ea0000-03ffe000	Deferred        d3d9
> PE	04000000-04053000	Deferred        dxva2
> PE	04060000-041ca000	Deferred        cryptui
> PE	041d0000-041ec000	Deferred        dhcpcsvc
> PE	041f0000-04205000	Deferred        vcruntime140
> PE	04210000-0426b000	Deferred        winex11
> PE	04a40000-04a7b000	Deferred        msv1_0
> PE	05000000-0518a000	Deferred        mfplat
> PE	1315b000-1d0df000	Export          libcef
> PE	1d0df000-1d217000	Deferred        chrome_elf
> PE	1dfcf000-1e3cb000	Export          battle.net helper
> PE	61c40000-61c67000	Deferred        userenv
> PE	61d40000-61d59000	Deferred        sys_msvcrt
> PE	622c0000-62357000	Deferred        rsaenh
> PE	62500000-628c7000	Deferred        oleaut32
> PE	62a00000-62a19000	Deferred        msmpeg2vdec
> PE	62c40000-62cc2000	Deferred        concrt140
> PE	62e40000-62e80000	Deferred        credui
> PE	63580000-6363d000	Deferred        oleacc
> PE	63880000-6391c000	Deferred        wintrust
> PE	63c00000-63d01000	Deferred        winhttp
> PE	64180000-64220000	Deferred        winspool
> PE	643c0000-6444c000	Deferred        ws2_32
> PE	646c0000-64737000	Deferred        iphlpapi
> PE	64a80000-64ab8000	Deferred        win32u
> PE	65200000-65213000	Deferred        esent
> PE	65680000-65911000	Deferred        msvcrt
> PE	65f00000-65f18000	Deferred        msauddecmft
> PE	66080000-66166000	Deferred        shlwapi
> PE	66480000-664b4000	Deferred        imagehlp
> PE	66640000-6665b000	Deferred        version
> PE	667c0000-66800000	Deferred        shcore
> PE	66fc0000-66fe4000	Deferred        compstui
> PE	67500000-67563000	Deferred        imm32
> PE	67bc0000-6803d000	Deferred        comctl32
> PE	68280000-682c1000	Deferred        jsproxy
> PE	684c0000-685fd000	Deferred        combase
> PE	68880000-68d71000	Deferred        user32
> PE	68e40000-68ec0000	Deferred        netapi32
> PE	69040000-69256000	Deferred        comdlg32
> PE	69580000-6960f000	Deferred        mfreadwrite
> PE	69840000-6992d000	Deferred        advapi32
> PE	6a1c0000-6a1d9000	Deferred        wevtapi
> PE	6a600000-6a629000	Deferred        kerberos
> PE	6a680000-6a6aa000	Deferred        wtsapi32
> PE	6aac0000-6adc4000	Deferred        ucrtbase
> PE	6b140000-6b16c000	Deferred        rtworkq
> PE	6bb40000-6bba7000	Deferred        propsys
> PE	6bbc0000-6bc51000	Deferred        sechost
> PE	6c100000-6c2c3000	Deferred        wininet
> PE	6c980000-6cc58000	Deferred        msvcp140
> PE	6d480000-6d512000	Deferred        secur32
> PE	6da80000-6dcbc000	Deferred        gdi32
> PE	6e200000-6e247000	Deferred        bcrypt
> PE	6e800000-6e821000	Deferred        ncrypt
> PE	6ecc0000-6ed00000	Deferred        dnsapi
> PE	6f240000-6f3b2000	Deferred        winmm
> PE	6f6c0000-6f6df000	Deferred        nsi
> PE	70000000-7026e000	Deferred        dbghelp
> PE	704c0000-704db000	Deferred        dwmapi
> PE	70880000-709c3000	Deferred        uiautomationcore
> PE	70d80000-70e24000	Deferred        uxtheme
> PE	71080000-710aa000	Deferred        hid
> PE	712c0000-71314000	Deferred        mpr
> PE	71400000-7208d000	Deferred        shell32
> PE	7a800000-7ac60000	Deferred        opengl32
> PE	7b000000-7b526000	Deferred        kernelbase
> PE	7b800000-7b96a000	Dwarf-4         kernel32
> PE	7bc00000-7bed9000	Dwarf-4         ntdll
> Threads:
> process  tid      prio    name (all IDs are in hex)
> 00000020 Battle.net.exe
> 	00000024    0     CrBrowserMain
> 	00000108    0     
> 	00000118    0     
> 	0000011c    0     
> 	00000120    0     
> 	00000124    0     
> 	0000012c    0     
> 	0000014c    0     wine_mmdevapi_notification
> 	00000150    0     
> 	00000154    0     
> 	00000170    0     
> 	00000174    0     
> 	00000198    0     
> 	0000019c    0     
> 	000001a0    0     
> 	000001a4    0     
> 	000001a8    0     ThreadPoolServiceThread
> 	000001ac    0     ThreadPoolForegroundWorker
> 	000001b0   -2     ThreadPoolBackgroundWorker
> 	000001b4    0     ThreadPoolForegroundWorker
> 	000001b8    0     Chrome_IOThread
> 	000001bc    0     MemoryInfra
> 	000001e0    0     
> 	000001fc   -2     ThreadPoolSingleThreadCOMSTASharedBackgroundBlocking0
> 	00000200    0     wine_threadpool_worker
> 	00000204    0     wine_threadpool_waitqueue
> 	00000210    0     ThreadPoolSingleThreadCOMSTASharedForeground1
> 	00000214    0     ThreadPoolSingleThreadCOMSTASharedForegroundBlocking2
> 	00000218    0     ThreadPoolForegroundWorker
> 	0000021c    0     ThreadPoolSingleThreadForegroundBlocking3
> 	00000220    0     CompositorTileWorker1
> 	0000022c    0     wine_sechost_device_notify
> 	00000230    0     VideoCaptureThread
> 	0000023c   -2     ThreadPoolSingleThreadSharedBackgroundBlocking4
> 	00000240    0     ThreadPoolSingleThreadSharedForegroundBlocking5
> 	00000244    0     ThreadPoolSingleThreadSharedForeground6
> 	00000248   -2     ThreadPoolBackgroundWorker
> 	0000024c    0     ThreadPoolForegroundWorker
> 	00000250    0     ThreadPoolForegroundWorker
> 	00000254    0     ThreadPoolForegroundWorker
> 	00000258    0     ThreadPoolForegroundWorker
> 	000002fc    0     wine_rpcrt4_server
> 	00000300    0     
> 	00000320    0     
> 	00000324   15     
> 	00000328    0     
> 	00000358    0     
> 	0000035c   15     
> 	00000360    0     
> 	00000368    0     
> 	0000036c    0     CacheThread_BlockFile
> 	00000404    0     wine_wininet_collect_connections
> 00000038 services.exe
> 	0000003c    0     
> 	00000040    0     wine_rpcrt4_server
> 	0000004c    0     wine_rpcrt4_io
> 	00000078    0     wine_rpcrt4_io
> 	000000a0    0     wine_rpcrt4_io
> 	000000bc    0     wine_rpcrt4_io
> 	000000d8    0     
> 	000000e4    0     wine_rpcrt4_io
> 	00000410    0     wine_threadpool_worker
> 00000044 winedevice.exe
> 	00000048    0     
> 	00000054    0     
> 	00000058    0     wine_sechost_service
> 	0000005c    0     
> 	00000068    0     
> 	0000006c    0     
> 	000000d0    0     
> 	000000d4    0     
> 00000060 explorer.exe
> 	00000064    0     
> 	0000009c    0     
> 	000000b8    0     wine_rpcrt4_server
> 00000070 winedevice.exe
> 	00000074    0     
> 	0000007c    0     
> 	00000080    0     wine_sechost_service
> 	00000084    0     
> 	00000088    0     
> 	0000008c    0     
> 	00000090    0     
> 00000094 plugplay.exe
> 	00000098    0     
> 	000000a4    0     
> 	000000a8    0     wine_sechost_service
> 	000000ac    0     wine_rpcrt4_server
> 	00000234    0     wine_rpcrt4_io
> 	00000238    0     wine_threadpool_worker
> 000000b0 svchost.exe
> 	000000b4    0     
> 	000000c0    0     
> 	000000c8    0     wine_sechost_service
> 000000dc rpcss.exe
> 	000000e0    0     
> 	000000e8    0     
> 	000000ec    0     wine_sechost_service
> 	000000f0    0     wine_rpcrt4_server
> 	000000f4    0     wine_rpcrt4_server
> 	000000f8    0     wine_rpcrt4_io
> 00000100 conhost.exe
> 	00000104    0     
> 0000013c Agent.exe
> 	00000140    0     
> 	0000015c    0     
> 	00000164    0     cURL Multi Loop
> 	00000168    0     Version Tasks
> 	0000016c    0     Size Estimator
> 	00000178    0     Pool 'Http Server' Worker #0
> 	0000017c    0     Pool 'Http Server' Worker #1
> 	00000180    0     Pool 'Http Server' Worker #2
> 	00000184    0     Pool 'Http Server' Worker #3
> 	00000188    0     Pool 'Http Server' Worker #4
> 	0000018c    0     Pool 'Http Server' Worker #5
> 	00000190    0     Pool 'Http Server' Worker #6
> 	00000194    0     Pool 'Http Server' Worker #7
> 	000001c0    0     Pool 'Http Server' Worker #8
> 	000001c4    0     Pool 'Http Server' Worker #9
> 	000001c8    0     Pool 'Http Server' Worker #10
> 	000001cc    0     Pool 'Http Server' Worker #11
> 	000001d0    0     Http Server Accept
> 	000001d4    0     Install Manager
> 	000001d8    0     Config ProcessFetchQueue
> 	000001dc    0     Ribbit Network Watchdog
> 	000001e4    0     Ribbit Config Fetcher
> 	000001e8    0     Operation Loop
> 	000001ec    0     Pool 'Concurrent Op' Worker #0
> 	000001f0    0     Pool 'Concurrent Op' Worker #1
> 	000001f8    0     
> 	000003c0    0     AIO Thread
> 	000003c4    0     TACT Task Thread
> 	000003c8    0     TACT Task Thread
> 	000003cc    0     TACT Curl Downloader Thread
> 	00000434    0     InstallActions bna
> 00000144 conhost.exe
> 	00000148    0     
> 00000224 Battle.net.exe
> 	00000228    1     CrGpuMain
> 	00000284    0     
> 	0000028c    0     BrokerEvent
> 	000002d0    0     GpuWatchdog
> 	000002d4    0     wined3d_cs
> 	000002d8    0     ThreadPoolServiceThread
> 	000002dc    0     ThreadPoolForegroundWorker
> 	000002e0   -2     ThreadPoolBackgroundWorker
> 	000002e4    1     Chrome_ChildIOThread
> 	000002e8    0     wine_threadpool_worker
> 	000002ec    0     wine_threadpool_waitqueue
> 	000002f0    0     ThreadPoolForegroundWorker
> 	000002f4    1     VizCompositorThread
> 	000002f8    0     ThreadPoolForegroundWorker
> 	00000380    0     ThreadPoolForegroundWorker
> 	00000384   -2     ThreadPoolBackgroundWorker
> 0000025c Battle.net.exe
> 	00000260    0     CrUtilityMain
> 	0000026c    0     
> 	00000270    0     BrokerEvent
> 	00000274    0     ThreadPoolServiceThread
> 	00000278    0     ThreadPoolForegroundWorker
> 	0000027c   -2     ThreadPoolBackgroundWorker
> 	00000280    0     Chrome_ChildIOThread
> 	00000288    0     ThreadPoolForegroundWorker
> 00000264 Battle.net.exe
> 	00000268    0     CrUtilityMain
> 	00000290    0     
> 	00000294    0     BrokerEvent
> 	00000298    0     ThreadPoolServiceThread
> 	0000029c    0     ThreadPoolForegroundWorker
> 	000002a0   -2     ThreadPoolBackgroundWorker
> 	000002a4    0     Chrome_ChildIOThread
> 	000002a8    0     ThreadPoolForegroundWorker
> 	000002ac    0     ThreadPoolForegroundWorker
> 	000002b0    0     wine_threadpool_worker
> 	000002b4   -2     ThreadPoolSingleThreadCOMSTASharedBackgroundBlocking0
> 	000002b8    0     wine_threadpool_waitqueue
> 	000002bc   -2     ThreadPoolBackgroundWorker
> 	00000378    0     CacheThread_BlockFile
> 00000370 (D) Z:\home\alles\Games\Battle.net\Battle.net.exe
> 	00000374    0 <== CrRendererMain
> 	00000388    0     
> 	0000038c    0     BrokerEvent
> 	00000390    0     ThreadPoolServiceThread
> 	00000394    0     ThreadPoolForegroundWorker
> 	00000398   -2     ThreadPoolBackgroundWorker
> 	0000039c    1     Chrome_ChildIOThread
> 	000003a0    0     wine_threadpool_worker
> 	000003a4    0     wine_threadpool_waitqueue
> 	000003a8    0     ThreadPoolForegroundWorker
> 	000003ac    0     GpuMemoryThread
> 	000003e8    0     
> 000003b8 conhost.exe
> 	000003bc    0     
> System information:
>     Wine build: wine-8.16
>     Platform: x86_64 (guest: i386)
>     Version: Windows 10
>     Host system: Linux
>     Host version: 6.9.6-amd64-desktop-rolling

How start Battle.net properly? It is possible?

Thanks for all answers.

**Comment by BLumia** [comment]
While I'm not good at triage wine issues, I personally suggest to try Valve's Proton instead of deepin-wine since Proton is more suitable for playing games. You can get [ProtonUp-Qt](https://flathub.org/apps/net.davidotek.pupgui2) or [Bottles](https://flathub.org/apps/com.usebottles.bottles) from flathub and try them instead :)

**Reply by ADalles** [reply]
Fine I try it. Thanks for your hint.
